docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport.js integration module
+
+{WFM-RC-NameShort} authentication and authorisation module using link:http://passportjs.org[PassportJS].
+The Passport.js module is the default security module for {WFM-RC-NameShort}.
+
+Authentication mechanisms, known as link:http://passportjs.org/docs/configure[strategies], are packaged as individual modules.
+Applications can choose which strategies to employ, without creating unnecessary dependencies.
+Passport.js offers comprehensive set of link:http://passportjs.org/docs/configure[strategies] that support authentication
+using a username and password, Facebook, Twitter, and more.
+This module allows the:
+
+- Authorisation
+- Authentication
+- Storing user sessions using link:https://github.com/expressjs/session[express-session]
+- Default strategy using cookies
+
+
+== Configuration of PassportAuth module
+As the PassportAuth module is the default security module it is install as part of running the Demo App.
+No configuration is required in order to use this module. It is an out of the box security solution for
+{WFM-RC-NameShort}. Passport.js is a lightweight solution that offers all the
+functionality required to get started. Passport.js has low overhead and setup costs.
+
+
+== Developer Customisation
+The link:./api/0.0.1/auth-passport/docs/modules/_src_auth_defaultstrategy_.html[default strategy] for the PassportAuth
```

Comment:
Full links don't appear to resolve 
http://raincatcher.feedhenry.io/api/0.0.1/auth-passport/docs/modules/_src_auth_defaultstrategy_.html
will leave as is till this issue is resolved.